code/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from 'styled-components';
+
+const HeaderContainer = styled.header `
+    padding: 6px 0 0 0;
+    margin: 20px auto;
+    @media (min-width: 768px) {
+        margin: 0;
+        }
```

Comment:
Two weeks ago I learned that you can write your media queries like this when working with styled components (maybe more useful if there are more styling). 
`const devices = {
    mobil: '(min-width: 375px)',
    tablet: '(min-width: 768px)',
    desktop: '(min-width: 1024px)'
}`
and then it is possible to only target the device you wish to change, for example

`@media ${devices.tablet} {
        font-size: 20px;
    }`